pkg/platform/local/platform.go

```diff
@@ -164,6 +164,12 @@ func (p *Platform) CreateFunction(createFunctionOptions *platform.CreateFunction
 		}
 	}
 
+	// if function exists, perform some validation with new function create options
```

Comment:
At this point the function config may not exist

Reply:
yes, see my comment above ^